tools/dm-task-config-argument-description.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Data Migration Task Configuration Options
+summary: This document introduces the configuration options that apply to Data Migration tasks.
+category: tools
+---
+
+# Data Migration Task Configuration Options
+
+This document introduces the configuration options that apply to Data Migration tasks. 
+
+## `task-mode`
+
+- String
+- The task mode of data migration to be executed 
+- Value: `full`/`incremental`/`all`
+
+    - `full`: Only makes a full backup of the upstream database and then makes a full import to the downstream database.
```

Comment:
how about `and then restore them to the downstream database`

Reply:
restore it?